### PR TITLE
Add Teraranger one to autodoc

### PR DESCRIPTION
**Pull request by meee1**

### CODE REVIEW

**Comment by rmackay9** [comment]
I guess all the GCSs will be ok with multiple items having the same value?  So in this case there are two items that both have the value "4"

**Comment by WickedShell** [comment]
Iirc other similar things use a comma for multiple items under the same value. But I think it could probably be handled either way on the GCS's.

**Comment by rmackay9** [comment]
Here's the problem though, when the user goes back in to see what range finder they are using the ground station will have no way to know which "4" to display.

File: libraries/AP_RangeFinder/RangeFinder.cpp [hunk 1]
```diff
@@ -29,7 +29,7 @@ const AP_Param::GroupInfo RangeFinder::var_info[] = {
     // @Param: _TYPE
     // @DisplayName: Rangefinder type
     // @Description: What type of rangefinder device that is connected
-    // @Values: 0:None,1:Analog,2:APM2-MaxbotixI2C,3:APM2-PulsedLightI2C,4:PX4-I2C,5:PX4-PWM,6:BBB-PRU,7:LightWareI2C,8:LightWareSerial
+    // @Values: 0:None,1:Analog,2:APM2-MaxbotixI2C,3:APM2-PulsedLightI2C,4:PX4-I2C,4:TeraRangerOne-I2C,5:PX4-PWM,6:BBB-PRU,7:LightWareI2C,8:LightWareSerial
```
Comment (magicrub) [inline libraries/AP_RangeFinder/RangeFinder.cpp]: Is the TeraRangerOne 100% compatible/clone with the px4-i2c?

Comment (meee1) [inline libraries/AP_RangeFinder/RangeFinder.cpp]: as far as im aware yes, the reason for the separation is so other paramaters like the min and max can be predefined.
although randy did bring up a  good point about gcs's possible wont know what to display if there are 2, 4's

**Pull request closed without merging**